### PR TITLE
Add preview card to be used by twitter.

### PR DESCRIPTION
**Pull request by b-abel**
Added configuration that causes Twitter to generate a "card" with
logo, title and a reference to the Hono Twitter account, when a link
to the Hono site appears in a Twitter message.

Signed-off-by: Abel Buechner-Mihaljevic <Abel.Buechner@bosch-si.com>

### CODE REVIEW

File: site/homepage/layouts/partials/head.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <meta name="robots" content="all,follow">
   <meta name="googlebot" content="index,follow,snippet,archive">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ .Title }}</title>
+  <title>{{ .Title }} :: {{ .Site.Title }}</title>
```
Comment (sophokles73) [inline site/homepage/layouts/partials/head.html]: is this related to the twitter stuff?

Comment (b-abel) [reply site/homepage/layouts/partials/head.html]: No. I consider this a general improvement, because it unifies the way how the title is build between homepage and documentation. But it should have been a separate commit/PR.

Comment (sophokles73) [reply site/homepage/layouts/partials/head.html]: exactly :+1:
can you please create a separate PR for it?

Comment (b-abel) [reply site/homepage/layouts/partials/head.html]: I removed the title configuration from this PR.
